### PR TITLE
Unable to zoom on layer

### PR DESCRIPTION
**Pull request by vlt1**
## Description
After investigating the problem in the issue it turned out that there are two things:
- The crs parsing of bounding box sometimes yields incorrect results. I've hopefully made it a bit more robust
- Even though the crs in bounding boxes of Geobretagne says EPSG:2154 the bounding boxes actually are in CRS84
(ESPG:4326 with (lon,lat) coordinate order). For this I've added a new service parameter BBOX CRS that will override the one
on bounding box in CSW record if present. It would have to be set for necessary csw catalog services on georchestra(like Geobretagne) after merging.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [x] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [ ] Other... Please describe:

## Issue
#5737 
[mapstore2-georchestra/#242](https://github.com/georchestra/mapstore2-georchestra/issues/242)

**What is the current behavior?**
#5737 

**What is the new behavior?**
Description

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/components/catalog/editor/AdvancedSettings/RasterAdvancedSettings.js [hunk 1]
```diff
@@ -29,5 +31,17 @@ export default ({
                     onChange={event => onChangeServiceFormat(event && event.value)} />
             </Col >
         </FormGroup>
+        {service?.type === 'csw' && <FormGroup style={{ display: 'flex', alignItems: 'center', paddingTop: 15}}>
+            <Col xs={6}>
```
[the remark this answers is not in the document]
Comment (allyoucanmap) [reply web/client/components/catalog/editor/AdvancedSettings/RasterAdvancedSettings.js:35]: If this select input is remove ignore my coment above

**Comment by allyoucanmap** [comment]
@vlt1 please wait feedback from @tdipisa before proceed

**Comment by tdipisa** [comment]
@allyoucanmap thank you so much for your review it is quite useful I think.
I completelly agree with  your points and the select input introduced for the catalog advanced settings must be removed.
As soon as it is removed we can merge the PR is all the other code is good.

File: web/client/epics/catalog.js [hunk 2]
```diff
@@ -372,8 +372,9 @@ export default (API) => ({
             .switchMap(({ text }) => {
                 const state = getState();
                 const pageSize = pageSizeSelector(state);
+                const selectedCatalog = selectedCatalogSelector(state);
                 const { type, url } = selectedCatalogSelector(state);
-                return Rx.Observable.of(textSearch({ format: type, url, startPosition: 1, maxRecords: pageSize, text }));
+                return Rx.Observable.of(textSearch({ format: type, url, startPosition: 1, maxRecords: pageSize, text, options: {service: selectedCatalog} }));
```
Comment (allyoucanmap) [inline web/client/epics/catalog.js:377]: This code seems related to the select input. Is it still needed?

**Review by allyoucanmap — APPROVED** [review]

**Pull request merged**